apps/web/components/apps/App.tsx

```diff
@@ -416,7 +420,7 @@ const InstallAppButtonChild = ({
     },
   });
 
-  if (!userAdminTeams?.length || appCategories.some((category) => ["calendar", "conferencing"].includes(category))) {
+  if (!userAdminTeams?.length || appCategories.some((category) => category === "calendar" || category === "conferencing" && !concurrentMeetings)) {
```

Comment:
Don't show the dropdown to install a conferencing app for a team/org if the `concurrentMeetings` prop isn't true.